SPECS/binutils/linker-script-readonly-keyword-support.patch

```diff
@@ -1,156 +0,0 @@
-From 6b86da53d5ee2022b9065f445d23356190380746 Mon Sep 17 00:00:00 2001
```

Comment:
This file still needs to be here. The bootstrap build of binutils is still version 2.37 and it requires it